### DISCUSSION TITLE
Default VLAN

### DISCUSSION THREAD

**Discussion by clemone210** [community-scripts/ProxmoxVE#393] [Q&A] [answered]
Hello together,

since my network setup is not ideal my default VLAN is 1 which grants access to the main network.
I move all my VM's and Containers into VLAN 11, which I define at the network hardware within each container or vm.
Is it possible to use the default settings of a script but still modify the VLAN?
Or is it possible at proxmox to set the default VLAN to each network?

**Comment by Mellowlynx** [comment, marked as answer]
No, this is currently only possible through the advanced settings.
I am not sure if this will change, but there is an idea out there to make this possible.
So it might change at some point.
